core/storage/base_model/gae_models.py

```diff
@@ -550,7 +558,24 @@ def get_snapshots_metadata(
                 model has been deleted. Default is False.
 
         Returns:
-            list(dics). Each dict representing a model snapshot.
+            list(dict). Each dict contains a snapshot metadata:
```

Comment:
Each dict contains metadata for a particular snapshot. It has the following keys: